### ISSUE TITLE
When use 2 countdowns, when first finished, it stops the second too.

### ISSUE THREAD

**Issue by yan87** [pughpugh/react-countdown-clock#41]
Hi,

I am new with React, I try use 2 countdown clocks, the problem when the first finished (onComplete), the second one stops. How can I run multiple react-countdown-clock?

**Comment by pughpugh** [comment]
Sorry for the delayed response. If this is still a problem, please post an example and I will take a look.

**Comment by zchauhan** [comment]
@pughpugh This is still an issue.

**Comment by pughpugh** [comment]
OK thanks @zchauhan , I will take a look.

**Comment by zchauhan** [comment]
@pughpugh 

The issue is with below code
```
_stopTimer: ->
    for timeout in @_timeoutIds
      clearTimeout timeout
```

**Comment by pughpugh** [comment]
Yup. Timeout IDs need to be stored in state, not the component definition. They are currently bleeding across instances. I think this is the cause of a number of problems at the moment.

**Comment by zchauhan** [comment]
Yes, You are right. I am wondering how props can be shared ?
Didn't experience this so far.

**Comment by dannykoshy** [comment]
Any update on this?

**Comment by Simek** [comment]
Bump! Any news or workaround?

**Comment by Simek** [comment]
I've manage to implement little, dirty w/o in which I've override start/stop methods and lift `timeoutIds` to component wrapper state:

```
export default class Counter extends ReactCountdownClock {
	constructor(props) {
		super(props);
		this.state = { tids: [] };

		this._stopTimer = () => {
			for (let timeout in this.state.tids) {
				clearTimeout(timeout);
			}
		};

		this._startTimer = () => {
			this.setState({ tids: setTimeout(this._tick(), 200)});
		};
	}
}
```

It's not pretty or solid code but works.

**Comment by dannykoshy** [comment]
Hey thanks,

I had solved it. Cheers!

**Comment by obrejla** [comment]
Hi all,

also bumped into this... It's really a blocker, is the fix planned?

Thanks a lot.
